tools/build.py

```diff
@@ -109,6 +109,8 @@ def devhelp(helpstring):
                         help='build unittests')
     parser.add_argument('-v', '--verbose', action='store_const', const='ON', default='OFF',
                         help='increase verbosity')
+    parser.add_argument('--vm-exec-stop', metavar='X', choices=['ON', 'OFF'], default='ON', type=str.upper,
```

Comment:
I think you also need to disable the `vm execution stopping` by default in here.